mirgecom/simutil.py

```diff
@@ -291,7 +287,7 @@ def check_naninf_local(discr, dd, field):
     return not np.isfinite(s)
 
 
-def compare_fluid_solutions(discr, red_state, blue_state):
+def compare_fluid_solutions(discr, red_state, blue_state, comm=None):
```

Comment:
I don't think we actually need `comm` being passed in, since it's not doing anything anyways (reductions infer the comm from `discr`)
```suggestion
def compare_fluid_solutions(discr, red_state, blue_state):
```